test/bdd/README.md

```diff
@@ -0,0 +1,52 @@
+# Behavior-Driven Development
+
+This is the root directory for behavior-driven tests illustrating the expectations of how features will work from the perspective of the end-user (via the browser).
+
+## Background
+
+As the project grows, we need to be able to ensure the user interface is behaving as expected when new functionality is added. Part of the process refinement was to "tests" as criteria for new pull requests. We are looking to implement those tests using `behave` and `behaving`. 
+
+For example, see [`operation-sanity`](https://github.com/cyverse/operation-sanity), the end-to-end test suite used when validating an Atmosphere release.
+
+## Setup
+
+Right now, tests are being written from a clone of this repository on a Mac (or Linux) machine that target other installed environments. We are not running these tests in a "headless", server-side mode yet (but could, and will, in the future). The notion here is that using the `behave` command from a virtualenv, we can execute and iterate on features without having to go through the "test-runner" cycle of Django or other tools. 
+
+Ensure that you have the Selenium driver for Firefox installed:
+```
+brew install geckodriver
```

Comment:
The story for getting `geckodriver` on Linux is not as nice as `<pkg-mgr> install` it seems. From what I found so far, everyone is point to the releases on GitHub: 

https://github.com/mozilla/geckodriver/releases

Will look a bit further and see if we _require_ the WebDriver + Marionette bits.